SocialCareCaseViewerApi/V1/Boundary/Response/CreateAllocationResponse.cs

```diff
@@ -0,0 +1,11 @@
+namespace SocialCareCaseViewerApi.V1.Boundary.Response
+{
+    public class CreateAllocationResponse
```

Comment:
This is identical to the request object. There's no need to return these same values, so this should be updated to return values relevant to the results of the action taken.